### PR TITLE
IPRO20 : data-base-schema

### PR DESCRIPTION
**Pull request by Cedricsakah5**

### CODE REVIEW

**Comment by lambertydev** [comment]
You need to follow the convention we agreed for the PR title

File: index.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ app.get('/', (req, res)=>{
     res.send('Hello world')
     
 })
-app.listen(5500, () => console.log('listening on port 5500...'))
+
+app.listen(3000, () => console.log('listening on port 5500...'));
```
Comment (lambertydev) [inline index.js]: Are you using port 5500 or 3000? I'm seeing 2 different ports here

File: models/user.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const User =  mongoose.model('User', new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        minlength: 3,
+        maxlength: 250
+
+    },
+    userType:{
+        enum: ['Owner', 'Customer'],
```
Comment (lambertydev) [inline models/user.js]: Better to use small letters here: ['owner', 'customer']

File: models/property.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        type: String,
+        required: true,
+        minlength: 3,
+        maxlength: 250
+
+    },
+    description:{
+        type: String,
+        required: true,
+        minlength: (5),
```
Comment (lambertydev) [inline models/property.js]: Any reason for putting the values of `minlength` and `maxlength` in braces?

File: models/property.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    ownerId: {
+        type: mongoose.Types.ObjectId,
+        ref: 'User',
+        required:true
+    },
+    image:{
+        type: Image,
+        required:true
+    }
+}));
```
Comment (lambertydev) [inline models/property.js]: Always try to add an empty line at the end of each file

File: models/rental.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    
+  userId:{
+        type: mongoose.Types.ObjectId, 
+        ref: 'User'
+
+    },
+
+    appartmentId:{
+        type: mongoose.Types.ObjectId, 
+        ref: 'Rental'
```
Comment (lambertydev) [inline models/rental.js]: Shouldn't ref be `Appartment` instead?

File: index.js (hunk 1 above)
Comment (Cedricsakah5) [reply index.js]: its a mistake sir i was trying to run the code on port 5500 but it was giving me an error so i decided to change the port to 3000 to see if i will work and forgot to edit the message at the log

File: models/user.js (hunk 2 above)
Comment (Cedricsakah5) [reply models/user.js]: ok sir

File: models/property.js (hunk 3 above)
Comment (Cedricsakah5) [reply models/property.js]: No sir .

File: models/rental.js (hunk 5 above)
Comment (Cedricsakah5) [reply models/rental.js]: yes, sir it should be Apartment mistake!

**Review by lambertydev — APPROVED** [review]

**Pull request merged**